src/Client.js

```diff
@@ -67,6 +67,19 @@ class Client extends EventEmitter {
     async initialize() {
         let [browser, page] = [null, null];
         
+        // eslint-disable-next-line no-useless-escape
+        const foldernameRegex = /^(?!.{256,})(?!(aux|clock\$|con|nul|prn|com[1-9]|lpt[1-9])(?:$|\.))[^ ][ \.\w-$()+=[\];#@~,&amp;']+[^\. ]$/i;
```

Comment:
What happens if you don't validate the name? I'm assuming an error is thrown elsewhere, what does that look like? This seems kind of os-dependent.

Reply:
I put all the OS naming rules on one regex

it throws an error if the name is invalid